terraform/modules/matrix-service/driver_lambda.tf

```diff
@@ -0,0 +1,85 @@
+resource "aws_iam_role" "matrix_service_driver_lambda" {
+  name = "matrix-service-driver-daemon-${var.deployment_stage}"
+
+  assume_role_policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+POLICY
+}
+
+resource "aws_iam_role_policy" "matrix_service_driver_lambda" {
+  name = "matrix-service-driver-daemon-${var.deployment_stage}"
+  role = "${aws_iam_role.matrix_service_driver_lambda.name}"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "LogsPolicy",
+      "Effect": "Allow",
+      "Resource": "arn:aws:logs:*:*:*",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ]
+    },
+    {
+      "Sid": "DynamoPolicy",
+      "Effect": "Allow",
+      "Action": [
+        "dynamodb:PutItem"
+      ],
+      "Resource": "arn:aws:dynamodb:us-east-1:861229788715:table/dcp-matrix-service-state-table-${var.deployment_stage}"
```

Comment:
please add account id to your environment variables + terraform variables and interpolate. Any hard coded account ids will make it difficult to deploy to another account (ie prod)

Reply:
Done